bundles/org.eclipse.equinox.launcher/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %pluginName
 Bundle-SymbolicName: org.eclipse.equinox.launcher;singleton:=true
-Bundle-Version: 1.6.400.qualifier
+Bundle-Version: 1.6.500.qualifier
```

Comment:
Why is this version incremented.  I don't see any changes to the launcher.

Reply:
the build complained about it

Reply:
Can you please try again without the extra version bumps?

Reply:
ok, removed it now it works.